clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -269,6 +269,10 @@ Changes in existing checks
   <clang-tidy/checks/modernize/loop-convert>` to support for-loops with
   iterators initialized by free functions like ``begin``, ``end``, or ``size``.
 
+- Improved :doc:`modernize-loop-convert
```

Comment:
Typo "dor" + leading spaces in second and third line,.
There is already entry for this check, just join this information with previous one.